word_guess.rb

```diff
@@ -0,0 +1,101 @@
+#https://github.com/Ada-C7/Word-Guess
+require 'faker'
+
+class WordGame
+
+  def initialize
+    @lives = 9
+    @word = Faker::Cat.unique.name.downcase
+    @underscore_array = Array.new( @word.length, "_" )
+    @word_array = @word.split("")
+
+  end
+
+  def convert_string_to_array
+    @word_array = @word.split("")
+    # puts "#{@underscore_array}"
+    # puts "#{@underscore_array.join(" ")}"
+    @underscore_string = @underscore_array.join(" ")
+    # puts "#{@word_array}"
+    # puts "#{@word_array.join("")}"
+    @word_string = @word_array.join("")
+  end
+
+def big_loop
+  print "Guess a letter or name: "
+  @user_input = gets.chomp
+
+
+  if @word_array.include? @user_input then
+    @word_array.each_with_index do |letter, index|
+
+      if letter == @user_input
+        @underscore_array[index] = letter
+      end
+
+      if @underscore_array.join("") == @word_array.join("")
+        puts "\nYou win!"
+        exit
+      end
+    end
+    ascii_art
+    puts "\n#{@underscore_array.join(" ")}"
+    big_loop
```

Comment:
I notice that you call `big_loop` from inside `big_loop` here, effectively using it as a way to get back to the top of the method. This technique, where a method calls itself, is called "recursion". Recursive methods can be really hard to reason about so we don't cover them until near the end of your CS Fun classes, but they can also do some really powerful things.

For this assignment, can you think of a way to solve the problem using a loop, possibly combined with those `break` and `next` statements we learned about in class?